packages/web/package.json

```diff
@@ -38,24 +38,29 @@
   "files": [
     "LICENSE.md",
     "README.md",
-    "dist/"
+    "CHANGELOG.md",
+    "dist/",
+    "bin/"
   ],
   "scripts": {
     "start": "npm run watch",
     "clean": "npx rimraf dist",
     "test": "true",
     "resources": "node ../../scripts/package-resources.mjs",
-    "build": "npm run clean && npm run resources && node scripts/build.mjs",
-    "watch": "npm run clean && npm run resources && node scripts/watch.mjs",
+    "changelog:create": "node ../../scripts/changelog/create.mjs",
+    "changelog:generate": "node ../../scripts/changelog/generate.mjs",
+    "build": "npm run clean && npm run resources && npm run changelog:generate && node scripts/build.mjs",
+    "watch": "npm run clean && npm run resources && npm run changelog:generate && node scripts/watch.mjs",
     "publish": "npm run build && npm publish --access public",
     "lint": "npx prettier --write ."
   },
   "dependencies": {
-    "onnxruntime-web": "^1.16.0",
     "@types/lodash": "^4.14.195",
     "@types/node": "^20.3.1",
     "lodash": "^4.17.21",
     "ndarray": "^1.0.19",
+    "onnxruntime-web": "^1.16.0",
+    "yaml": "^2.3.4",
```

Comment:
Is yaml a real dependency or a devDependency?

Reply:
Good catch